code/game/machinery/cloning.dm

```diff
@@ -186,11 +186,12 @@
 	H << "<span class='notice'><b>Consciousness slowly creeps over you as your body regenerates.</b><br><i>So this is what cloning feels like?</i></span>"
 
 	H.hardset_dna(ui, se, H.real_name, null, mrace, features)
-	H.faction |= factions
+	if(H)

```

Comment:
In case someone changes the record se value to monkey one Probably better way would be rewrite hardset to return new mob or cloner to handle monkeys properly.